### DISCUSSION TITLE
How Can/Should WZDx Cover "Special" Traveled Way Closures (e.g., sidewalks, bike lanes, shared use paths, etc.)?

### DISCUSSION THREAD

**Discussion by RHolt-RIDOT** [usdot-jpo-ode/jpo-wzdx#120] [Ideas]
USDOT/FHWA have been advocates for years for ensuring the needs of ALL traveler modes/means along the nations roadways are met.  My admittedly-quick glance at the current (V2) specification shows me that the focus seems to be on generic lane/road closures, without any specific standardization or convention for how to codify / account for otherwise "special" surface traveled ways such as those noted in the Subject line of this Post, in particular sidewalks which are most common.  Forgive me if I am just missing it and/or this is already being discussed by those more heavily involved with WZDx (thank you for all of your efforts!), and if so please call me out.  But if not I (and I assume USDOT/Volpe/consultant) would welcome seeing any thoughts/ideas on this here. 

 As comments on recent Federal transportation rulemakings have shown, there's many advocates for improved ped / bike mobility and traveler information out there, so I think it would be great if there's a way to standardize, via the WZDx effort, just how things should be codified "for them".  If by chance you feel it's more important to just focus on how WZDx data will benefit CAVs for now, just recall the last time you were walking along an urban street and came upon a completely barricaded closure of your path, with no clear or alternate way to go, and how you felt.  I understand baby steps are needed with all of this, but will be grateful if we can somehow show via these standardization efforts that non-motorized travelers have not been forgotten and that a means to disseminate data about work zone closures that sometimes affect only them (e.g., a sidewalk closure) has been included.  

Again, if others are working on or have ironed this out already please don't be bashful about calling me out and forgive me (I am unfortunately not quite yet an actual user or spec'er of the WZDx specification).  And I do see some areas of the current specification that seem open enough to potentially use even if/when the IOO is not necessarily dealing with a traditional roadway lane closure.  But if this is the starting discussion on this then I thank you for reading and hope you can help push this along.

**Comment by Roads1** [comment]
Being a recent new person to the use and needs of WZDX, I agree with RHolt-RIDOT. For all State and local ROW owners that are required to follow federal and local jurisdiction laws, regulations and policies regarding to the ADA (American Disabilities Act) needs. This would be a great tool to add to Work Zone needs for pedestrian and related impacts. In the WZDX "objects", the closest item would be the "WorkZoneRoadEvent". Many of the core details for that have a high chance to confuse a 3rd party that utilizes the WZDX feed to supplement their user provided information. A WorkZoneSidewalk/PathEvent needs to be created to help fix this need. Many of the "WorkZoneRoadEvent" (AKA RoadEventCoreDetails) "core details" could be left as is as an information base. The exceptions would be any items that refers to a **road** impact such as "vehicle impact" or "lanes". This will also assist in a State 511 mapping systems to identify locations for those who may use it for pedestrian or bicycle navigation needs.
